kapitan/inputs/copy.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+#
+# Copyright 2019 The Kapitan Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import shutil
+
+from kapitan.inputs.base import InputType, CompiledFile
+from kapitan.resources import inventory
+
+logger = logging.getLogger(__name__)
+
+
+class Copy(InputType):
+    def __init__(self, compile_path, search_paths, ref_controller):
+        super().__init__("copy", compile_path, search_paths, ref_controller)
+
+    def compile_file(self, file_path, compile_path, ext_vars, **kwargs):
+        """
+        Write items in path as plain rendered files to compile_path.
+        path can be either a file or directory.
+        kwargs:
+            reveal: default False, set to reveal refs on compile
+            target_name: default None, set to current target being compiled
+        """
+        reveal = kwargs.get('reveal', False)
+        target_name = kwargs.get('target_name', None)
+
+        try:
+            logging.debug("Copying {} to {}.".format(file_path, compile_path))
+            if os.path.isfile(file_path):
+                if os.path.isfile(compile_path):
+                    shutil.copy2(file_path, compile_path)
+                else:
+                    os.makedirs(compile_path, exist_ok=True)
+                    shutil.copy2(file_path, compile_path + "/" +
```

Comment:
Avoid using harcoded path separators as they may differ between different operating systems.
Use `os.path.join` instead https://www.geeksforgeeks.org/python-os-path-join-method/

Reply:
Changed it everywhere to be os.path.join.